CHANGELOG.md

```diff
@@ -21,6 +21,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 -->
 
+## [2.0.0-rc.2] - 2024-11-22
+
+### Added
+
++ Emit CMake warnings for MSVC + CUDA incompatibilities introduced by MSVC 1940 & 1941 ([#1225](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1225))
++ Add visualisation support for Environment Directed Graphs ([#1239](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1239))
```

Comment:
This feature was also in that PR
https://github.com/FLAMEGPU/FLAMEGPU2/pull/1239/commits/e574f2ede68c6430a3917449cbf4b95e914fe680

```suggestion
+ Add visualisation support for Environment Directed Graphs ([#1239](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1239))
+ Add visualisation support for setting the initial camera roll angle ([#1239](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1239))
```